### ISSUE TITLE
Source not found

### ISSUE THREAD

**Issue by VlaD-T** [softvis-research/Getaviz#131]
Would be nice to see an error alert, if source was not found. For example I've tried to create a new visualization with a link from "Usage" page. File is not available at the moment "**501 HTTPS Required**", so the generator couldn't generate the model. 

Just after seeing the log and trying to access the file manually it was clear what happened.

**Comment by naraesk** [comment]
Yes, there is a lot to do with regards to robustness and meaningful error messages
